service/history/workflow/mutable_state_state_status.go

```diff
@@ -24,12 +24,12 @@ func setStateStatus(
 	case enumsspb.WORKFLOW_EXECUTION_STATE_CREATED:
 		switch state {
 		case enumsspb.WORKFLOW_EXECUTION_STATE_CREATED:
-			if status != enumspb.WORKFLOW_EXECUTION_STATUS_RUNNING {
+			if status != enumspb.WORKFLOW_EXECUTION_STATUS_RUNNING && status != enumspb.WORKFLOW_EXECUTION_STATUS_PAUSED {
```

Comment:
same Q re. if a new execution can be in paused state.

Reply:
No. Will update it.